axelrod/strategies/graaskamp.py

```diff
@@ -0,0 +1,86 @@
+import random
+from axelrod.action import Action
+from axelrod.player import Player
+from scipy.stats import chisquare
+
+C, D = Action.C, Action.D
+
+
+class Graaskamp(Player):
+    """
+    Play Tit For Tat for the first 50 rounds;
+	Defects on round 51;
+	Plays 5 further rounds of Tit For Tat;
+	A check is then made to see if the opponent is playing randomly
+	in which case it defects for the rest of the game;
+	The strategy also checks to see if the opponent is playing Tit For Tat.
```

Comment:
This is a bit incomplete from https://github.com/Axelrod-Python/Axelrod/issues/1109 see:

> Step 5 is a bit tricky as it refers to a strategy called "Analogy". I have not been able to find any information describing this strategy. I suggest that anyone who tries to work on this does their best to find that information but if there is non available let us simply ignore that part of the check but clearly point that out in the documentation of the strategy.

Have you found any information about `analogy`? If not we should very clearly point this out in the documentation.